cli/relevance_tests/works/test_precision.py

```diff
@@ -145,6 +145,16 @@
         expected_ids=["rtdee482"],
         description="Case-insensitive partial titles",
     ),
+    PrecisionTestCase(
+        search_terms="EPB/ENCY/9.v1",
+        expected_ids=["ctuhg29m"],
+        description="Exact match on slash separated shelfmark",
+    ),
+    PrecisionTestCase(
+        search_terms="FTY.S.AI",
+        expected_ids=["cb9mq2d2"],
+        description="Exact match on dot separated shelfmark",
+    ),
```

Comment:
Should we have a test reflecting the [cross_fields/OR](https://github.com/wellcomecollection/catalogue-api/blob/f4a1e1357a4f1d197d84e075a6aec95651cc4391/search/src/main/resources/WorksQuery.json#L68) behaviour?

e.g. provide both of the search terms from these two, and expect both of the results.

IIUIC, there is a desire to be able to put in a bunch of identifiers and get them all back.  I suspect there may also be a desire to put in an identifier and a term from the title together, and still get them back even if they don't correspond to the same document (or one has a typo).

Reply:
Yes, good spot.

Reply:
I've added tests for multiple shelfmarks in a query, as well as mixing IDs and shelfmarks. Mixing title terms and IDs is not supported in the current search and I think is a change to be considered separately?